bindings/php/opendal-php.stubs.php

```diff
@@ -20,6 +20,111 @@
 
 // Stubs for opendal-php
 
-namespace {
-    function debug(): string {}
+namespace OpenDAL {
+    class Operator {
+        public function __construct(string $scheme_str, array $config) {}
+
+        /**
+         * Write bytes into given path.
+         */
+        public function write(string $path, string $content): mixed {}
+
+        /**
+         * Read the whole path into bytes.
+         */
+        public function read(string $path): string {}
+
+        /**
+         * Check if this path exists or not.
+         */
+        public function is_exist(string $path): int {}
```

Comment:
Maybe we should add docs about it's meaning: `0` for `false` and `1` for `true`?

Reply:
done